prism/__init__.py

```diff
@@ -1,8 +1,8 @@
-from prism.prism import Prism, load_schema, create_table, upload_file
```

Comment:
I see the functions `load_schema()`, `create_table()`, and `upload_file()` were dropped in your suggested changes.  Those functions were essential in my mind as not all users want to go through every single step needed just to create a table or upload a file.  What are your thoughts behind removing these wrapper functions that simplify the process of interacting with the API?

Reply:
I was thinking about that today and was wondering about those routines in the prism class and when they would be used.  I was leaning towards wrapping those activities exclusively in the CLI.  However if we expect the prism class to be part of other projects, I can quickly refactor the code for loading files that I have in the tables, buckets and dataChange commands into callable functions similar to the original upload function.  This is a minor refactor where the CLI commands would simply call the functions in the prism class.  Note: each of my upload routines is doing different sanity checks so I'd likely end up with more than one function to handle uploads.  I'll review the original functions to ensure no features/capabilities are lost.